tools/db_bench_tool.cc

```diff
@@ -4956,8 +4967,7 @@ class Benchmark {
 
     RandomGenerator gen;
     WriteBatch batch(/*reserved_bytes=*/0, /*max_bytes=*/0,
-                     FLAGS_write_batch_protection_bytes_per_key,
-                     user_timestamp_size_);
+                     0 /* protection_bytes_per_key */, user_timestamp_size_);
```

Comment:
Why do we need to zero these?

Reply:
It's not needed, I've reverted this change back. I was trying out ways to exclude write batch checksum calculation outside of calculation for ops/sec in db_bench, but this was not the way to do it.